samples/databus/custom-serializer/Core_6/Shared/JsonDataBusSerializer.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Text;
+
+namespace Shared
+{
+    using System.IO;
+    using Newtonsoft.Json;
+    using NServiceBus.DataBus;
+
+    #region CustomDataBusSerializer
+
+    public class JsonDataBusSerializer : IDataBusSerializer
+    {
+        public void Serialize(object databusProperty, Stream stream)
+        {
+            using (var writer = new StreamWriter(stream, Encoding.UTF8, bufferSize: 1024, leaveOpen: true))
```

Comment:
Will that look odd inside a `using` statement? Should I remove this into a separate method? `CreateNonClosingStreamWriter()`?